### DISCUSSION TITLE
Updates when booting swiss from Disk

### DISCUSSION THREAD

**Discussion by kern2011** [emukidid/swiss-gc#596] [Q&A] [answered]
I've read it is possible for swiss (on a disk) to automatically load a newer version from the SD card on boot. I've also seen people say it's not possible. So is it possible, and if so, how do you set it up?

**Comment by Extrems** [comment, marked as answer]
Simply put the Swiss DOL as `boot.dol` on the default device seen in the file browser.

**Comment by RexIsGod** [comment]
It doesn't seem possible to reset to the Swiss menu when you do this though.
